<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PandaRPG Engine Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="animated">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Tools</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Texturing</string>
        </property>
        <item>
         <property name="text">
          <string>add texture to ground</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>clear textures</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Objects</string>
        </property>
        <item>
         <property name="text">
          <string>add object with texture in tile</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Lightning</string>
        </property>
        <item>
         <property name="text">
          <string>add Light</string>
         </property>
        </item>
       </item>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="texturesTab">
        <attribute name="title">
         <string>Textures</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="texturesFilter">
           <property name="placeholderText">
            <string>filter string</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Image preview...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="texturePool">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideLeft</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerItem</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::Fixed</enum>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="charactersTab">
        <attribute name="title">
         <string>Characters</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="characterPool"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_2"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave_Scene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWorkspace">
    <property name="title">
     <string>Workspace</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAdvanced_featues"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuShaders">
    <property name="title">
     <string>Shaders</string>
    </property>
    <addaction name="actionPPL"/>
    <addaction name="actionToonShading"/>
    <addaction name="actionAmbientOcclusion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWorkspace"/>
   <addaction name="menuShaders"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionScene">
   <property name="text">
    <string>Scene</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave_Scene">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAdvanced_featues">
   <property name="text">
    <string>Visit website...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPPL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PerPixel Lighting</string>
   </property>
  </action>
  <action name="actionToonShading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toon Shading</string>
   </property>
  </action>
  <action name="actionAmbientOcclusion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ambient Occlusion</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
